src/ControlPanel.jsx

```diff
@@ -106,6 +106,7 @@ class ControlPanel extends Component {
 ControlPanel.propTypes = {
   relay: propTypes.element.isRequired,
```

Comment:
just wondering, where is this defined?

Reply:
I'm not sure, I copied it when I refactored Map and ControlPanel into separate files